src/EnergyPlus/WeatherManager.cc

```diff
@@ -341,12 +341,6 @@ namespace Weather {
         static constexpr std::string_view EnvDSTYFormat("Environment:Daylight Saving,Yes,{},{},{}\n");
         static constexpr std::string_view DateFormat("{:02}/{:02}");
         static constexpr std::string_view DateFormatWithYear("{:02}/{:02}/{:04}");
-        std::string StDate;
-        std::string EnDate;
-        int DSTActStMon;
-        int DSTActStDay;
-        int DSTActEnMon;
-        int DSTActEnDay;
```

Comment:
[src/EnergyPlus/WeatherManager.cc:345]:(style),[variableScope],The scope of the variable 'EnDate' can be reduced.
[src/EnergyPlus/WeatherManager.cc:346]:(style),[variableScope],The scope of the variable 'DSTActStMon' can be reduced.
[src/EnergyPlus/WeatherManager.cc:347]:(style),[variableScope],The scope of the variable 'DSTActStDay' can be reduced.
[src/EnergyPlus/WeatherManager.cc:348]:(style),[variableScope],The scope of the variable 'DSTActEnMon' can be reduced.
[src/EnergyPlus/WeatherManager.cc:349]:(style),[variableScope],The scope of the variable 'DSTActEnDay' can be reduced.